packages/story-editor/src/components/panels/design/sizePosition/radius.js

```diff
@@ -32,7 +32,7 @@ import { useCommonObjectValue, focusStyle } from '../../shared';
 import { MULTIPLE_DISPLAY_VALUE, MULTIPLE_VALUE } from '../../../../constants';
 import Tooltip from '../../../tooltip';
 
-const DEFAULT_BORDER_RADIUS = {
+export const DEFAULT_BORDER_RADIUS = {
```

Comment:
Should this go somewhere else? If so, where?

Reply:
Maybe either to `elements/constants.js` or to `constants/index.js` directly.